### PR TITLE
feature/PEER-978 Unique URI of pagination pages

### PR DESCRIPTION
**Pull request by alexeishulga**

### CODE REVIEW

File: app/containers/App/index.js [hunk 1]
```diff
@@ -327,6 +339,16 @@ const App = ({
             }
           />
 
+          <Route
+            path={'/experts/page:paginationPage'}
```
Comment (AweAwe95) [inline app/containers/App/index.js:343]: same as above

File: app/containers/App/index.js [hunk 2]
```diff
@@ -210,6 +210,8 @@ const App = ({
             <Route path={routes.feed(':communityid')} render={(props) => Wrapper(Feed, props)} />
           )}
 
+          <Route path={'/feed/page:paginationPage'} render={(props) => Wrapper(Feed, props)} />
```
[the remark this answers is not in the document]
Comment (alexeishulga) [reply app/containers/App/index.js]: done: https://github.com/peeranha/peeranha-web/pull/1437/commits/edb32d469df2605397e81453a502fc41103208c5

File: app/containers/App/index.js [hunk 3]
```diff
@@ -317,6 +319,16 @@ const App = ({
             }
           />
 
+          <Route
+            path={'/discussions/page:paginationPage'}
```
[the remark this answers is not in the document]
Comment (alexeishulga) [reply app/containers/App/index.js]: done: https://github.com/peeranha/peeranha-web/pull/1437/commits/edb32d469df2605397e81453a502fc41103208c5

File: app/containers/App/index.js (hunk 1 above)
Comment (alexeishulga) [reply app/containers/App/index.js]: done: https://github.com/peeranha/peeranha-web/pull/1437/commits/edb32d469df2605397e81453a502fc41103208c5

File: app/containers/App/index.js [hunk 4]
```diff
@@ -338,6 +360,11 @@ const App = ({
             render={(props) => Wrapper(Questions, { ...props, postsTypes: [POST_TYPE.tutorial] })}
           />
 
+          <Route
+            path={'/tutorials/page:paginationPage'}
```
[the remark this answers is not in the document]
Comment (alexeishulga) [reply app/containers/App/index.js]: done: https://github.com/peeranha/peeranha-web/pull/1437/commits/edb32d469df2605397e81453a502fc41103208c5

**Review by AweAwe95 — APPROVED** [review]

**Review by YuriyLu — APPROVED** [review]

**Pull request merged**